doc/architecture/database_guidelines.md

```diff
@@ -0,0 +1,20 @@
+# Database Guidelines
+
+This document outlines the conventions and guidelines for designing and maintaining a well-structured and organized database. Please follow these guidelines to ensure consistency and ease of use across the database.
+## 1. Table Names
+
+**Plural**: Table names should be plural to accurately represent the multiple rows of data they contain. For example, use users instead of user.
```

Comment:
I don't think we should give negative examples (how not to do it), simply give examples how it should be done, e.g.:

```suggestion
**Plural**: Table names should be plural to accurately represent the multiple rows of data they contain. For example: `users` and plants`.
```